frontend/src/components/layout/RightColumn/RightColumn.tsx

```diff
@@ -37,17 +37,25 @@ export default component$(() => {
 		{ iconName: 'fa-globe', linkText: 'Federated', linkTarget: '/public', linkActiveRegex: /^\/public\/?$/ },
 	]
 
+	const aboutLink = { iconName: 'fa-ellipsis', linkText: 'About', linkTarget: '/about', linkActiveRegex: /^\/about/ }
+
 	return (
-		<div class="flex flex-col justify-between right-column-wrapper text-slate-400">
+		<div class="bg-slate-800 xl:bg-transparent flex flex-col justify-between right-column-wrapper text-slate-400">
 			<div>
-				<div class="p-4">
-					<a class="no-underline flex items-center" href="https://mastodon.social">
+				<div class="xl:p-4">
+					<a class="no-underline hidden xl:flex items-center" href="https://mastodon.social">
 						<WildebeestLogo size="small" />
-						<span class="text-white font-bold text-xl ml-[-27px] mt-[-27px]">ildebeest</span>
+						{/* TODO: We need to move the text inside the logo component for better reusability
```

Comment:
@jculvey I can do this in a followup PR is not a big deal, but I was wondering, do we need to manually add the text ourselves? wasn't/shouldn't it be included in the svg?